docs/BR.md

```diff
@@ -1756,153 +1754,1152 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `policyIdentifier` |              | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                 |
+| \ \ **2+**                 | SHOULD       | The CA SHOULD include additional Reserved Certificate Policy Identifiers for each type of Subscriber certificate that may be issued beneath it. |
+| \ \ \ \ `policyIdentifier` | MUST         | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                 |
+| \ \ **3+**                 | MAY          | The CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                 |
+| \ \ **4**                  | MUST NOT     | The CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
+
+##### 7.1.2.10.5 Certificate Policies - Non-Affiliated CA
+
+The following requirements apply to the Certificate Policies extension within CA certificates that are issued to and operated by a CA that is an not an Affiliate of the Issuing CA.
+
+If present, the Certificate Policies extension MUST be formatted as follows:
+
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` |              | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **2+**                 | SHOULD       | The CA SHOULD include additional Reserved Certificate Policy Identifiers for each type of Subscriber certificate that may be issued beneath it. |
+| \ \ \ \ `policyIdentifier` | MUST         | A Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **3+**                 | MAY          | The CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
```

Comment:
The BRs currently say (section 7.1.2.2 Subordinate CA Certificate) "The following fields MAY be present if the Subordinate CA is not an Affiliate of the entity that controls the Root CA.". Did we go from "MAY" to "SHOULD NOT" with a specific reason?

Reply:
This is from RFC 5280. Recall that "RECOMMENDED" == "SHOULD" and "NOT RECOMMENDED" == "SHOULD NOT" (that is, they are interchangable)

From https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.4
```
   To promote interoperability, this profile RECOMMENDS that policy
   information terms consist of only an OID.
```

It is semantically equivalent to read this as "SHOULD NOT" contain a `policyQualifier`

Reply:
Yes, it is clearer now. Thanks.